ansible/roles/ocp4-workload-jdg-workshop/tasks/workload.yml

```diff
@@ -46,8 +46,8 @@
     -e CHE_URL=https://codeready-codeready.{{ route_subdomain }}
     -e KEYCLOAK_URL=https://keycloak-codeready.{{ route_subdomain }}
     -e ROUTE_SUBDOMAIN={{ route_subdomain }}
-    -e CONTENT_URL_PREFIX='https://raw.githubusercontent.com/RedHat-Middleware-Workshops/dg8-workshop/master/docs'
-    -e WORKSHOPS_URLS='https://raw.githubusercontent.com/RedHat-Middleware-Workshops/dg8-workshop/master/docs/{{ workshop_labs_url }}'
+    -e CONTENT_URL_PREFIX='https://raw.githubusercontent.com/RedHat-Middleware-Workshops/dg8-workshop/ocp-4.4/docs'
+    -e WORKSHOPS_URLS='https://raw.githubusercontent.com/RedHat-Middleware-Workshops/dg8-workshop/ocp-4.4/docs/{{ workshop_labs_url }}'
```

Comment:
shouldn't it be a set depending on  the value of `ocp4_installer_version` ?

Something like:
```yaml
"{{ ocp4_install_version | regex_replace('^([0-9])\\.([0-9]*).*', '\\1.\\2')  }}"
```

Reply:
no, this is our own branch version, so its how we manage our guides on top. the installer version does not effect this.